jenkins-plugin/src/main/java/org/jenkinsci/plugins/pipeline/maven/publishers/InvokerRunsPublisher.java

```diff
@@ -31,6 +31,9 @@
 import hudson.model.Run;
 import hudson.model.StreamBuildListener;
 import hudson.model.TaskListener;
+import hudson.tasks.junit.JUnitResultArchiver;
+
+import org.apache.commons.lang3.BooleanUtils;
```

Comment:
Careful with this dependency if it is not being picked up via an explicit dep on the `commons-lang3-api` library plugin. Trivial uses like this are best avoided. https://github.com/jenkinsci/jenkins/pull/6267

Reply:
Rewrote using vanilla java